flows/common/common.go

```diff
@@ -68,36 +80,84 @@ func Handler(fn hdrsFunc, match matchFunc, reporter *Reporter) lwotg.FlowGenerat
 			size := len(buf.Bytes())
 
 			klog.Infof("%s Tx interface %s", flow.Name, tx)
-			handle, err := pcap.OpenLive(tx, 1500, true, pcapTimeout)
+
+			ih, err := pcap.NewInactiveHandle(tx)
+			if err != nil {
+				klog.Errorf("cannot create handle, err: %v", err)
+				return
+			}
+			defer ih.CleanUp()
+			if err := ih.SetImmediateMode(true); err != nil {
+				klog.Errorf("cannot set immediate mode on handle, err: %v", err)
+				return
+			}
+
+			if err := ih.SetPromisc(false); err != nil {
+				klog.Errorf("cannot set promiscous mode, err: %v", err)
+				return
+			}
+
+			if err := ih.SetSnapLen(packetBytes); err != nil {
+				klog.Errorf("cannot set packet length, err: %v", err)
+			}
+
+			// Set buffer to be 200MB of sent packets.
```

Comment:
how is this number chosen? does this correspond to some particular number of packets we are confident is above what we'd need?

Reply:
It is, and was, a terrible choice :-) Reverted back to the default.